src/tiktok_uploader/upload.py

```diff
@@ -351,14 +353,16 @@ def _set_video(driver, path: str = '', num_retries: int = 3, **kwargs) -> None:
                 )
 
             # An exception throw here means the video failed to upload an a retry is needed
-            WebDriverWait(driver, config['explicit_wait']).until(upload_confirmation)
+            WebDriverWait(driver, config['uploading_wait']).until(upload_confirmation)
 
             # wait until a non-draggable image is found
             process_confirmation = EC.presence_of_element_located(
                 (By.XPATH, config['selectors']['upload']['process_confirmation'])
                 )
             WebDriverWait(driver, config['explicit_wait']).until(process_confirmation)
```

Comment:
By the way, What this part of the code if for ? I feel like it is not useful or maybe it is just on my computer ?

Reply:
I think this is just the button that pops up after the upload is successful. So you _could_ click it but it's not necessary. Doesn't look like anything depends on it so was probably just here as a placeholder?

Reply:
Oh I see I feel like we might consider remove it. I removed this part in my fork because it broke too much and removing it did not change anything it just made it better (but it might be only my computer)